pipeline/backend/kubernetes/utils.go

```diff
@@ -65,23 +65,23 @@ func isImagePullBackOffState(pod *v1.Pod) bool {
 	return false
 }
 
-// getClientOutOfCluster returns a k8s clientset to the request from outside of cluster.
+// getClientOutOfCluster returns a k8s client set to the request from outside of cluster.
 func getClientOutOfCluster() (kubernetes.Interface, error) {
-	kubeConfigPath := os.Getenv("KUBECONFIG")
+	kubeConfigPath := os.Getenv("KUBECONFIG") // cspell:words KUBECONFIG
```

Comment:
Common, could be added to dictionary.

Reply:
![image](https://github.com/woodpecker-ci/woodpecker/assets/24977596/da9be8fe-74d2-4756-9474-f65a9b2e83eb)

not realy common i would argue